packages/synapse-react-client/src/lib/containers/SynapseNavDrawer.tsx

```diff
@@ -190,8 +188,7 @@ export const SynapseNavDrawer: React.FunctionComponent<
       ],
     },
     {
-      enabled:
-        currentUserBundle?.isACTMember || currentUserBundle?.isARReviewer,
```

Comment:
This was redundant. All ACT members are AR reviewers ([backend code](https://github.com/Sage-Bionetworks/Synapse-Repository-Services/blob/32429a031d70806a467a663c6262672f0c6a6495/services/repository-managers/src/main/java/org/sagebionetworks/repo/manager/dataaccess/DataAccessAuthorizationManagerImpl.java#L97))